Engineering/Frontend/WAI-ARIA.md

```diff
@@ -0,0 +1,70 @@
+---
+tags: engineering/frontend, wai-aria, accessibility
+author: Nguyen Dinh Nam
+date: 2022-09-30
+---
+
+Since modern websites and applications contain huge amounts of dynamic content and complex components with no semantics to describe what they mean, users with **no mouse** equiped or those dependent on technologies such as **[Screen Reader](https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn/related?hl=en)** might not be able to interact with those components.
```

Comment:
A bit weird intro :kek: Using a computer with no mouse is not usual, or touch devices don't need mouse. I think we can just state directly the role of WAI-ARIA: to assist users with disabilities.

For users that like to navigate the web without using the mouse, I think it's just a matter of preference.

Reply:
This intro and a few below paraphrases mention **Screen Reader** but it's only one of wai-aria targets. It should aim to **Assistive Technologies** like screen readers, text-to-speech software, alternate input technologies...